relbench/datasets/amazon.py

```diff
@@ -21,8 +21,10 @@
 
 class AmazonDataset(RelBenchDataset):
     name = "rel-amazon"
-    val_timestamp = pd.Timestamp("2014-01-01")
+    val_timestamp = pd.Timestamp("2015-06-01")
     test_timestamp = pd.Timestamp("2016-01-01")
```

Comment:
Shouldn't we make the gap between val and test also 365 // 4?
Also, let's confirm if this does not break the other tasks.

Reply:
Fixed!

Reply:
We also need to fix this for the other Amazon tasks, make them `365 // 4`.

In general, how to enforce this?